api/tests/support/support.h

```diff
@@ -729,6 +729,32 @@ class WaitCondition
     useconds_t timeoutDuration_;
 };
 
+class PollCondition
+{
+public:
+    PollCondition(int maxCount) : pollCount(0), pollMaxCount(maxCount)
```

Comment:
Probably acceptable to have these braces on the same line as the declaration, so the whole thing is just one line.